app/jobs/count_group_recent_activity_job.rb

```diff
@@ -0,0 +1,26 @@
+class CountGroupRecentActivityJob < ActiveJob::Base
+  queue_as :low_priority
+
+  def query
+    "UPDATE groups SET
+     recent_activity_count =
+      (5* (SELECT count(discussions.id)
+           FROM discussions
+           WHERE discussions.group_id = groups.id
+           AND discussions.created_at > current_date - 30)) +
+      (5 * (SELECT count(motions.id)
```

Comment:
it's made up. a starting point for fine tuning. I'm guessing that motions are 5 times more important than comments.